<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoverCreatorWidget</class>
 <widget class="QWidget" name="MoverCreatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>605</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Mover Creator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Designer::MoverCreatorTabWidget" name="mTabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>5</number>
       </property>
       <widget class="QWidget" name="mIntroTab">
        <attribute name="title">
         <string>Start/Setup</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1" colspan="3">
          <widget class="Designer::StartDialog" name="mStartDialog" native="true"/>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mEngineTab">
        <attribute name="title">
         <string>Engine</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="Designer::EngineDesignerWidget" name="mEngineDesignerWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mGeometryTab">
        <attribute name="title">
         <string>Geometry</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="Designer::GeometryWidget" name="mGeometryWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mAerodynamicsTab">
        <attribute name="title">
         <string>Aerodynamics</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="Designer::AerodynamicsWidget" name="mAerodynamicsWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mPerformanceTab">
        <attribute name="title">
         <string>Performance</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="Designer::PerformanceWidget" name="mPerformanceWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mControlsTab">
        <attribute name="title">
         <string>Pilots/Controls</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="Designer::ControlsWidget" name="mControlsWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mAutopilotTab">
        <attribute name="title">
         <string>Autopilot</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="Designer::AutopilotWidget" name="mAutopilotWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="mFlightTestTab">
        <attribute name="title">
         <string>Flight Test</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="Designer::FlightTestWidget" name="mFlightTestWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Designer::GeometryWidget</class>
   <extends>QWidget</extends>
   <header>GeometryWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::ControlsWidget</class>
   <extends>QWidget</extends>
   <header>ControlsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::AerodynamicsWidget</class>
   <extends>QWidget</extends>
   <header>AerodynamicsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::PerformanceWidget</class>
   <extends>QWidget</extends>
   <header>PerformanceWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::EngineDesignerWidget</class>
   <extends>QWidget</extends>
   <header>EngineDesignerWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::StartDialog</class>
   <extends>QWidget</extends>
   <header>StartDialog.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::MoverCreatorTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">MoverCreatorTabWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::AutopilotWidget</class>
   <extends>QWidget</extends>
   <header>AutopilotWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Designer::FlightTestWidget</class>
   <extends>QWidget</extends>
   <header>FlightTestWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
